SPDX-FileCopyrightText: 2021 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-2.0-or-later

mes libc + setting locale = not worky.

--- bash-2.05b/locale.c	2021-01-15 09:38:55.729307629 +1100
+++ bash-2.05b/locale.c	2021-01-15 11:19:01.929391346 +1100
@@ -190,7 +190,7 @@
 set_lang (var, value)
      char *var, *value;
 {
-  return ((lc_all == 0 || *lc_all == 0) ? setlocale (LC_ALL, value?value:"") != NULL : 0);
+  return 0;
 }
 
 /* Get the value of one of the locale variables (LC_MESSAGES, LC_CTYPE) */
